docs/src/pages/[platform]/connected-components/account-settings/delete-user/react.mdx

```diff
@@ -1,2 +1,164 @@
-## Delete User
-TODO
+import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
+import { DELETE_USER, OVERRIDES, WARNING } from './props';
+
+### Props
+
+<ReactPropsTable props={DELETE_USER} />
+
+### Basic Usage
+
+`DeleteUser` has `onSuccess` handler that will be called after successful user account deletion. You may use this callback to run any custom handling, such as navigating route or rendering confirmation messages.
+
+```jsx
+import React from 'react';
+import { DeleteUser } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+  
+  return (
+    <DeleteUser onSuccess={handleSuccess}/>
+  );
+}
+```
+
+
+### Override default delete handler
+You can override the default deleteUser handler with `handleDelete` prop to add any custom cleanup handlers before or after account deletion.
+
+```jsx
+import React from 'react';
+
+import { Auth } from 'aws-amplify';
+import { DeleteUser } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+
+  const handleDelete = async () => {
+    console.log("Doing some clean up...");
+    console.log("Done!");
+    await Auth.deleteUser();
+  };  
+      
+  return (
+    <DeleteUser onSuccess={handleSuccess}/>
+  );
+}
+```
+
+### Component Overrides
+
+You can provide your own UI components to override parts of `DeleteUser`. It supports the following slots:
+
+<ReactPropsTable props={OVERRIDES} />
+
+
+
+#### Reusing default components
+
+Default components are accessible through `DeleteUser.COMPONENT_NAME` (e.g. `DeleteUser.NewPassword`) for your convenience. This is useful if you're interested in modifying just a small part of UI instead of overriding the whole component.
+
+```jsx
+function App() {
+  const components = {
+    // reuse default component
+    SubmitButton: (props) => (
+      <DeleteUser.SubmitButton>Custom Submit Button</DeleteUser.SubmitButton>
+    ),
+  }
+
+  return (
+    <DeleteUser components={components} />
+  );
+}
+```
+
+#### Using your own component
+
+You can also provide your own custom to override components. 
```

Comment:
```suggestion
You can also provide your own custom buttons to override components. 
```

Reply:
or `elements` ?

Reply:
Oh, good catch, thanks! I'll go with

```suggestion
You can override parts of the `DeleteUser` component by passing in a `components` prop.
```

that I stole from initial FileUploader doc 😈